docker/es/justfile

```diff
@@ -42,3 +43,22 @@ NO_COLOR := "\\033[0m"
     just ../../_loop \
     '"$(just docker/es/check-index {{ index }} {{ dest }})" != "1"' \
     "Waiting for index '{{ index }}' to be ready..."
+
+# Check that the given index has the given number of docs
+@check-count index="image" count="5000":
```

Comment:
This looks quite similar to the above `wait-for-index` commands - do you think it makes sense to consolidate some logic?

Reply:
Merged in 0fc5e55.